core/src/main/scala/cats/UnorderedFoldable.scala

```diff
@@ -98,9 +98,13 @@ trait UnorderedFoldable[F[_]] extends Serializable {
     unorderedFoldMap(fa)(a => if (p(a)) 1L else 0L)
 }
 
+trait LowPriorityImplicits {
```

Comment:
```suggestion
private trait UnorderedFoldableLowPriority {
```

Reply:
I saw (late) that it couldn't be used later if it was private: `private trait UnorderedFoldableLowPriority escapes its defining scope as part of type cats.UnorderedFoldableLowPriority `
I changed it to protected, I think it makes sense, what do you think?